kieserver/image.yaml

```diff
@@ -200,6 +200,9 @@ envs:
   - name: "KIE_SERVER_DISABLE_KC_VERIFICATION"
     example: "true"
     description: "Disable Maven KIE Jar verification. It is recommended to test the kjar manually before disabling this verification."
+  - name: "KIE_SERVER_JBPM_CLUSTER"
+    example: "true"
+    description: "Enable the JPM clustering for Kie-Server fail-over, if less than two replica as confiured one additionl replica will be added"
```

Comment:
```suggestion
    description: "Enable the JPM clustering for Kie-Server for processes fail-over. If the number of replicas is less than two, one additional replica will be started."
```

Reply:
Hey @desmax74, can you please point me to where this condition is implemented? will it be added on the operator or on the kieserver image?

Reply:
@spolti https://github.com/kiegroup/kie-cloud-operator/pull/589/files#diff-29ad51b906bc49dd54db094a3b626e8e4a04628d03562e6c0d45db0e391ac2fbR705

Reply:
ok, thanks.